Bottom: a23d8585160d52d515b3e735117844e06b92388d
Top:    2fdd3f361429ecedb0940e6776397645fd1e9967
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 03:49:11 +0100

Refresh of try-to-avoid-warnings-from

---

diff --git a/src/parse_y.y b/src/parse_y.y
index ba31cd6..62fcc4b 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -131,7 +131,6 @@ static Coord new_units (PLMeasure m);
 
 parse
 		: parsepcb
-		| parsedata
 		| parsefont
 		| error { YYABORT; }
 		;
@@ -240,25 +239,6 @@ parsepcb
 		  }
 		;
 
-parsedata
-		:	{
-					/* reset flags for 'used layers';
-					 * init font and data pointers
-					 */
-				int	i;
-
-				if (!yyData || !yyFont)
-				{
-					Message(_("illegal fileformat\n"));
-					YYABORT;
-				}
-				for (i = 0; i < MAX_LAYER + 2; i++)
-					LayerFlag[i] = false;
-				yyData->LayerN = 0;
-			}
-		 pcbdata
-		;
-
 pcbfont
 		: parsefont
 		|
@@ -706,7 +686,6 @@ pcbdefinition
 		| rats
 		| layer
 		| element
-		| error { YYABORT; }
 		;
 
 via
